substrate/client/network/src/behaviour.rs

```diff
@@ -52,6 +52,8 @@ pub struct Behaviour<B: BlockT> {
 	peer_info: peer_info::PeerInfoBehaviour,
 	/// Discovers nodes of the network.
 	discovery: DiscoveryBehaviour,
+	/// Connection limits.
+	connection_limits: libp2p::connection_limits::Behaviour,
```

Comment:
nit: Should this be the first field of the `Behavior` struct? Since it would avoid calling into protocol, peer_info, discovery when a connection is limited?

Reply:
Done, not sure if matters much though